docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -194,6 +196,60 @@ command completion, or environment variables are presented throughout the
 Ansible Collection Testing because the act of installing the stable release of
 Ansible containing `ansible-test` is expected to setup those things for you.
 
+.. _meta_runtime_yml:
+
+meta directory
+--------------
+
+A collection can store some additional metadata in a ``runtime.yml`` file in the collection's ``meta`` directory. The ``runtime.yml`` file supports the top level keys:
+
+- *requires_ansible*:
+
+  The version of Ansible required to use the collection.
```

Comment:
Would be a good idea to mention you can specify multiple versions separated by a comma, e.g. `requires_ansible: '>=2.10,<2.11'`

Reply:
That's a good idea

Reply:
We should probably mention somewhere that this is a PEP440 Version Specifier set under the covers (https://www.python.org/dev/peps/pep-0440/#version-specifiers), and that we deviate from the PEP440 behavior by truncating prerelease segments from the Ansible version before running it through the compatibility sieve (so eg, Ansible 2.11.0b1 is compatible with something that lists >=2.11, even thought PEP440 says it's not).